cozy.example.yaml

```diff
@@ -140,6 +140,7 @@ jobs:
   # List of available workers:
   #
   #   - "clean-clients":     delete unused OAuth clients
+  #   - "index":             send data to a RAG for being indexed
```

Comment:
I'm wondering if it's not worth explicitly name it RAG, since the generic "index" terminology might be confusing with "traditional" index.

Reply:
I have hesitated, but I think the worker could easily evolve to allow other kinds of indexation (like a full-text index in meilisearch), so I kept a generic name

Reply:
Ok, fair enough

Reply:
Maybe we should stick to these generic terms and not talk avec RAGs at all except in the `ai` module documentation then.